pasteit/views/pasted.ejs

```diff
@@ -34,6 +35,12 @@
                 <div id="container" class="clearfix">
                     <div id="main-container">
                         <div id="paste-container">
+                            <div id="paste-header">
+                                <p>
+                                    <a id="delete" href="/delete/" style="float: right;">Delete</a>
```

Comment:
Inline styling needs to be moved to a CSS file.

Reply:
done in 25e11d9